Tests/Functional/Controller/ValueControllerTest.php

```diff
@@ -1,6 +1,7 @@
 <?php
+
 /*
- * This file is part of the Sulu CMS.
+ * This file is part of Sulu.
```

Comment:
we've just decided to add style-ci to product-bundle, which will reformat headers automatically in future.